packages/video_player/video_player_platform_interface/CHANGELOG.md

```diff
@@ -1,27 +1,15 @@
-## 4.0.0-nullsafety.1
+## 4.0.0
 
+* **Breaking Changes**:
+  * Migrate to null-safety
+  * Update to latest Pigeon. This includes a breaking change to how the test logic is exposed.
 * Add note about the `mixWithOthers` option being ignored on the web.
-
-## 4.0.0-nullsafety.0
-
-* Update to latest Pigeon.
-  This includes a breaking change to how the test logic is exposed.
-
-## 3.0.0-nullsafety.3
-
-* `messages.dart` sets Dart `2.12`.
-
-## 3.0.0-nullsafety.2
-
-* Bump Dart SDK to support null safety.
-
-## 3.0.0-nullsafety.1
-
 * Make DataSource's `uri` parameter nullable.
+* `messages.dart` sets Dart `2.12`.
 
-## 3.0.0-nullsafety
+## 3.0.0
```

Comment:
i'm confused. should this just be 3.0.0 breaking change? did we break a prerelease breaking change? 😅

Reply:
Yeah, `4.0.0-nullsafety.0` was published to pub. I doubt pub will let me go back to 3.0.0 :/